src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -489,6 +490,20 @@ public void testGetUriByFilenameFindsRelativeResourceOnClasspathPrefix() throws
         assertEquals("Checker", config.getName(), "Unexpected config name!");
     }
 
+    @Test
+    public void testIsCodePointWhitespaceTrue() {
+        final int[] codePoints = " 123".codePoints().toArray();
+        assertThat(CommonUtil.isCodePointWhitespace(codePoints, 0))
+                .isTrue();
+    }
+
+    @Test
+    public void testIsCodePointWhitespaceFalse() {
+        final int[] codePoints = " 123".codePoints().toArray();
+        assertThat(CommonUtil.isCodePointWhitespace(codePoints, 1))
+                .isFalse();
```

Comment:
This two tests are indentical on input data please combine them to one method.

Reply:
Done.